src/app/upravit-akci/[id]/page.tsx

```diff
@@ -16,7 +17,10 @@ const EditEventPage: NextPage<EditEventPageProps> = async ({ params }) => {
       <h1 className="mt-24 px-20 text-center text-2xl font-medium leading-normal md:mt-14 md:px-0 xl:text-start xl:text-4xl">
         Upravit sportovní událost
       </h1>
-      <EventForm event={event} />
+      <EventForm
+        event={event}
+        dateTimeIso={format(new Date(event?.date), "yyyy-MM-dd'T'HH:mm:ss'Z'")}
```

Comment:
Leave out the seconds. I wonder how it works from the server side. What is being sent when the server is in Frankfurt (or whatever) and you will be in a different timezone? Will it complain that there is a server-client mismatch? 

Just curious, not a biggie

Reply:
It needs to be there otherwise server will respond with 400 😮

Reply:
Wait what? If you leave out the second, it returns 400? Kinda fishy. How come you did not investigate it further?

Reply:
If it would be "yyyy-MM-dd'T'HH:mm'Z'" or even empty string... BE would reject it with "bad request" - 400. The weird thing is that it would still creare an event but it would miss date, name and description (other data would be there).